printersmng/src/main/java/org/qw3rtrun/p3d/printersmng/PrintersmngApplication.java

```diff
@@ -0,0 +1,13 @@
+package org.qw3rtrun.p3d.printersmng;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class PrintersmngApplication {
```

Comment:
We need to add Web-Flux controller as an API to manipulate with connections descriptions. And I would probably recommend you to add e2e tests for this module.

Reply:
Postman collection's requests as well